Index: OpenAI_API_Key
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>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
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/OpenAI_API_Key b/OpenAI_API_Key
--- a/OpenAI_API_Key	(revision 60d72793a0d8c3a07e450a9fd648860f985471c5)
+++ b/OpenAI_API_Key	(date 1712169740010)
@@ -1,1 +1,1 @@
-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
\ No newline at end of file
+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
\ No newline at end of file
Index: test
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test b/test
new file mode 100644
--- /dev/null	(date 1712170163163)
+++ b/test	(date 1712170163163)
@@ -0,0 +1,8 @@
+curl https://api.openai.com/v1/chat/completions \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer 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" \
+  -d '{
+     "model": "gpt-3.5-turbo",
+     "messages": [{"role": "user", "content": "Say this is a test!"}],
+     "temperature": 0.7
+   }'
\ No newline at end of file
